sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -1495,6 +1495,53 @@ pi_result piDevicesGet(pi_platform Platform, pi_device_type DeviceType,
   return PI_SUCCESS;
 }
 
+// sub-sub-device
+// TODO: do we need to gather Ordinals or just the SubSubDevicesCount will do ?
```

Comment:
I think we should collect ordinals, since spec does not guarantee all compute-groups come sequential.
You probably don't need to return total count (one can always ask vector.size())